data/docs/migration/migrate-from-honeycomb/dashboards.mdx

```diff
@@ -0,0 +1,95 @@
+
+### Creating Custom Dashboards
+
+In case there is no existing dashboard template that you can use, you can create a custom dashboard in SigNoz. Please follow [Manage Dashboards in SigNoz documentation](/docs/userguide/manage-dashboards/#steps-to-create-a-custom-dashboard/) for more details.
+
+## SigNoz Dashboard Features
+
+### Panel Types and Visualization Options
+
+When recreating your Honeycomb boards in SigNoz, it's important to understand which visualization types are available. SigNoz supports most common panel types and provides additional options beyond Honeycomb's capabilities:
+
+| Panel Type | Honeycomb | SigNoz | Notes |
+|------------|-----------|--------|-------|
+| Time Series | ✓ | ✓ | Both platforms support time series visualization |
+| Table | ✓ | ✓ | Both support tabular data display |
+| Bar Chart | - | ✓ | Enhanced visualization option in SigNoz |
+| Pie Chart | - | ✓ | Enhanced visualization option in SigNoz |
+| Value/Single Stat | - | ✓ | Single value display with thresholds |
+| Heatmap | ✓ | - | Use histogram metrics as alternative |
+| Text Panel | ✓ | - | Use dashboard descriptions or annotations |
+| SLO Panel | ✓ | - | Recreate using custom alert rules |
+
+For panels that don't have a direct equivalent in SigNoz, consider how you might achieve similar functionality using available visualization types. For example, Honeycomb's SLO panels can be replaced with custom alert rules and value panels showing SLO compliance metrics.
+
+Please refer to the [Panel Types](https://signoz.io/docs/dashboards/panel-types/) documentation for more details on the panel types supported by SigNoz.
+
+### Dashboard Features
+
+SigNoz supports essential dashboard features including:
+
+**Variables**: Create template variables for dynamic dashboards  
+**Query Builder**: [The Query Builder](https://signoz.io/docs/userguide/query-builder-v5/) is SigNoz's structured query interface that combines the power of expression-based querying with intelligent auto-completion  
+**Filters**: Apply filters across panels and data sources  
+**Sharing**: Share dashboards with team members and external users  
+**Time Controls**: Flexible time range selection and auto-refresh  
+
+For detailed dashboard management, refer to the [Manage Dashboards](https://signoz.io/docs/userguide/manage-dashboards/) documentation.
+
+
+## Honeycomb to SigNoz Functionality Mapping
+
+### Basic Query Translation
+
+Honeycomb uses proprietary query syntax while SigNoz uses standard PromQL for metrics and ClickHouse SQL for logs/traces. You can use SigNoz's [Query Builder](https://signoz.io/docs/userguide/query-builder-v5/) for visual query construction or write queries directly. Below are the most common query patterns and their SigNoz equivalents:
+
+| Honeycomb Query | SigNoz Equivalent | Notes |
+|----------------|-------------------|-------|
+| `COUNT` | `sum(rate(metric[5m]))` | PromQL rate function |
```

Comment:
I doubt if we should show promQL queries here.
@srikanthccv What can be best way to represent SigNoz equivalent here?

Reply:
Honeycomb treats everything as event and they don't have good metrics support. The equivalent is just QB Equivalent of logs and traces not promql.

Reply:
Should I remove this table completely then?

Reply:
No, we have equivalent for the same in SigNoz as well. For example `COUNT` in honeycomb is `count()` in SigNoz. Please check the logs query builder in SigNoz and you can update this table.